back-end/Service.py

```diff
@@ -0,0 +1,280 @@
+    mail.send_mail(order)
+    return jsonify(or_handler.add_new_order(order))
+
+@app.route('/getOrders')
+def get_orders():
+    db = DataAccessOrders()
+    or_handler = OrderHandler(db)
+    return jsonify(or_handler.get_orders())
+
+@app.route('/getLocationsInfo')
+def get_locations_info():
+    db = DataAccessLocation()
+    loc_info_handler = LocationHandler(db)
+    return jsonify(loc_info_handler.get_all_locations_info())
+
+@app.route('/getLocations')
+def get_locations():
+    db = DataAccess()
+    loc_info_handler = LocationHandler(db)
+    return jsonify(loc_info_handler.get_all_locations())
+
+
+
+
+@app.route('/getLocationTypes')
+def get_location_types():
+    db = DataAccess()
+    loc_info_handler = LocationHandler(db)
+    return jsonify(loc_info_handler.get_all_locations_types())
+
+@app.route('/addLocation', methods=['POST'])
+def add_location():
+    data = json.loads(request.stream.read())
+    location = Location(data["name"],data["type"],data["latitude"],data["longitude"], data["description"] )
+    loc_DB = DataAccessLocation()
+    loc_handler = LocationHandler(loc_DB)
+    loc_handler.add_location(location)
+    print(data)
+    image = Image(data["sphereImageList"][0]['name'], "sphere", data["type"], data["name"])
+    im_DB = DataAccessImage()
+    im_handler = ImageHandler(im_DB)
+    im_handler.add_image(image)
+    return '200'
+
+@app.route('/addLocationType', methods=['POST'])
+def add_location_type():
+    location_type = json.loads(request.stream.read())
+    loc_DB = DataAccessLocation()
+    loc_handler = LocationHandler(loc_DB)       
+    return  jsonify(loc_handler.add_location_type(location_type))
+
+
+@app.route('/sendMail', methods=['POST'])
+def send_mail():
+    data = json.loads(request.stream.read())
+    order = Order(data["date"], data["time"], data["name"], data["telefon"], data["email"], data["location"], data["eventType"], data["note"])    
+    return 
+ 
+
+@app.route('/upsertLink', methods=['POST'])
+def upsert_link():
+    data = json.loads(request.stream.read())
+    db = DataAccessLocation()
+    link_handler = ImageLinkerHandler(db)
+    link =  ImageLinker(data["origin"], data["destination"], data["latitude"], data["longitude"])
+    return jsonify(link_handler.upsert_link(link))
+
+@app.route('/getLinksToImage', methods=['POST'])
+def get_links():
+    data = json.loads(request.stream.read())
+    db = DataAccessLocation()
+    link_handler = ImageLinkerHandler(db)
+    image = Image(data["name"],data["imageType"],data["eventType"],data["location"])
+    return jsonify(link_handler.get_links_for_image(image))
+
+@app.route('/deleteLink', methods=['POST'])
+def delete_link():
+    data = json.loads(request.stream.read())
+    db = DataAccessLocation()
+    link_handler = ImageLinkerHandler(db)
+    link =  ImageLinker(data["origin"], data["destination"], data["latitude"], data["longitude"])
+    return jsonify(link_handler.delete_link(link))
+
+if __name__ == "__main__":

```

Comment:
I thought we removed this from here when I've added the support for digital ocean?